### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by frgm**

### CODE REVIEW

**Comment by frgm** [comment]
I don't have installed pandoc on my system.
would it be ok if I'll change only markdown files (if i can spot another typos)?

**Comment by sustrik** [comment]
Yes, sure.

File: hdata.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The data pointer in case of success. In case of failure, `NULL` is returned and
 ## ERRORS
 
 * `EDADF`: Invalid handle.
-* `ENOTSUP`: Provided type parameter doesn't match the typo of the handle.
```
Comment (paulofaria) [inline hdata.md]: Hahaha this one is ironic! Sorry had to comment. 😊

**Comment by sustrik** [comment]
HTML regenerated.

**Pull request merged**